src/components/forms/validations.jsx

```diff
@@ -23,6 +23,7 @@ module.exports.validations = {
         return phoneNumberUtil.isValidNumber(parsed);
     }
 };
+module.exports.validations.notEqualsUsername = module.exports.validations.notEquals;
```

Comment:
Oh right the error messages.  There's somewhere else where we hit this issue... it'd be nice if `validations` was more flexible, maybe by optionally taking a list of objects so you could have many of the same type of validations, with different messages for each.